smart-connector-rest-server/src/main/resources/openapi-sc.yaml

```diff
@@ -216,6 +216,18 @@ paths:
                   prefixes:
                     rdf: "http://www.w3.org/1999/02/22-rdf-syntax-ns#"
                     ex: "http://example.org/"
+              ASK with knowledge gaps enabled:
+                value:
+                  knowledgeInteractionType: AskKnowledgeInteraction
+                  graphPattern: "?a <http://example.org/isRelatedTo> ?b ."
+                  knowledgeGapsEnabled: "true"
+                description:
```

Comment:
Hey @JackJackie, just tuning in unasked 😄 . I like this description, but maybe move it from the example to the actual definition of the property? Not sure if it is possible, but similar to something like [this](https://github.com/TNO/knowledge-engine/blob/d482621eb38f7c0f243110c2ab55ca671444f02a/smart-connector-rest-server/src/main/resources/openapi-sc.yaml#L559)?

Reply:
I replaced the description to the head of the request body part.